solr/solrj/src/java/org/apache/solr/client/solrj/io/Tuple.java

```diff
@@ -276,14 +282,12 @@ public void setMetrics(Map<String, Map<?,?>> metrics) {
 
   public Tuple clone() {
     Tuple clone = new Tuple();
-    clone.fields.putAll(fields);
-    // TODO This doesn't copy EOF/Exception https://issues.apache.org/jira/browse/SOLR-15480
+    clone.putAll(fields);
```

Comment:
maybe shorten to `Tuple clone = new Tuple(this.fields);` here?

Reply:
Sure! Should I shorten it all the way down to a one line `return new Tuple(this.fields);`? I'm not sure what the general style is like around here, but I generally like doing assignment then return vs just return, mostly for debugging purposes, but I'm fine either way.